### PR TITLE
fixed for 50 table dfgen2 limit

### PR DESCRIPTION
**Pull request by m-kovalsky**

### CODE REVIEW

File: src/sempy_labs/migration/_create_pqt_file.py [hunk 1]
```diff
@@ -119,89 +139,90 @@ def __init__(
...
+            for t_name, query in table_map.items():
                 queryMetadata.append(
-                    QueryMetadata(e_name, None, None, None, False, False)
+                    QueryMetadata(t_name, None, None, None, True, False)
                 )
+            for e_name, v in expr_map.items():
+                e_kind = v[0]
+                if e_kind == "M":
+                    queryMetadata.append(
+                        QueryMetadata(e_name, None, None, None, True, False)
```
Comment (eisber) [inline src/sempy_labs/migration/_create_pqt_file.py:162]: nit: I'd suggest to use named parameters to improve readability

File: src/sempy_labs/migration/_create_pqt_file.py [hunk 2]
```diff
@@ -119,89 +139,90 @@ def __init__(
...
+
+            def obj_to_dict(obj):
+                if isinstance(obj, list):
+                    return [obj_to_dict(e) for e in obj]
+                elif hasattr(obj, "__dict__"):
+                    return {k: obj_to_dict(v) for k, v in obj.__dict__.items()}
+                else:
+                    return obj
+
+            jsonContent = json.dumps(obj_to_dict(rootObject), indent=4)
```
Comment (eisber) [inline src/sempy_labs/migration/_create_pqt_file.py:181]: why dumps and not just dump?

with open('data.txt', 'w') as out_file:
     json.dump(json_data, out_file, sort_keys = True, indent = 4,
               ensure_ascii = False)

File: src/sempy_labs/migration/_create_pqt_file.py [hunk 3]
```diff
@@ -60,54 +60,74 @@ def create_pqt_file(
...
+                    self.EngineVersion = EngineVersion
+                    self.QueriesMetadata = QueriesMetadata
+                    self.QueryGroups = QueryGroups
+
+            # STEP 1: Create MashupDocument.pq
+            mdfileName = "MashupDocument.pq"
+            mdFilePath = os.path.join(subFolderPath, mdfileName)
+            sb = "section Section1;"
+            for t_name, query in table_map.items():
+                sb = f'{sb}\nshared #"{t_name}" = '
```
Comment (eisber) [inline src/sempy_labs/migration/_create_pqt_file.py:130]: triple-quote + https://docs.python.org/3/library/textwrap.html for improved readability?

**Review by eisber — APPROVED** [review]

File: src/sempy_labs/migration/_create_pqt_file.py (hunk 1 above)
Comment (m-kovalsky) [reply src/sempy_labs/migration/_create_pqt_file.py:162]: fixed

File: src/sempy_labs/migration/_create_pqt_file.py (hunk 2 above)
Comment (m-kovalsky) [reply src/sempy_labs/migration/_create_pqt_file.py:181]: i think the code seems simple enough

File: src/sempy_labs/migration/_create_pqt_file.py (hunk 3 above)
Comment (m-kovalsky) [reply src/sempy_labs/migration/_create_pqt_file.py:130]: this seems clear(er) to me

**Pull request merged**